src/common/class.wpes-wp-post-field-builder.php

```diff
@@ -534,6 +534,127 @@ public function get_mappings( $args = array() ) {
 						),
 					),
 				),
+				'attachment' => array(
```

Comment:
I think we want this to be a nested object so that multiple images on a post do not get mixed together for things like color.

Also I am thinking that this should go into the image mapping, and we turn that into a nested object. There are a number of cases where an image is not necessarily attached to the post in question (esp when the same image is in multiple posts, but also images hosted elsewhere).